packages/truffle-debugger/lib/data/selectors/index.js

```diff
@@ -510,7 +679,61 @@ const data = createSelectorTree({
 
         words => (words || []).map(word => DecodeUtils.Conversion.toBytes(word))
       )
-    }
+    },
+
+    //HACK WARNING
+    //the following selectors depend on solidity.next
+    //do not use them when the current instruction is a context change!
+
+    /**
+     * data.next.scope
+     * this should probably be named data.next.node!
```

Comment:
let's deprecate it, then! we don't have a `data.next.node`, right? We can just make that selector and use `data.next.scope` to be a deprecated alias.

Reply:
Cool, I'll do that then!